examples/middleware.rs

```diff
@@ -25,8 +25,8 @@ impl UserDatabase {
 // application state. Because it depends on a specific request state,
 // it would likely be closely tied to a specific application
 fn user_loader<'a>(
```

Comment:
similarly to the graphql example, UserDatabase could just have a derive(Clone) and nothing else would need to change